### DISCUSSION TITLE
Closing access to the web panel from the outside via ufw-docker for Amnezia Wg

### DISCUSSION THREAD

**Discussion by Ogurec007** [wg-easy/wg-easy#837] [Q&A] [answered]
[Repo](https://github.com/spcfox/amnezia-wg-easy)

Hello, I'm trying to block access to the web panel via ufw-docker, but nothing works.  The main task is that without AWG I cannot get to the page http://0.0.0.0:51821

I am using the following command - `ufw-docker allow amnezia-wg-easy`

Next I write - `ufw route allow proto tcp from 10.8.0.0/24 to any port 51821`, but the access remained from outside...

**Comment by pheiduck** [comment, marked as answer]
amnezia-wg-easy seems to be a frok which we don't provide support for.
